cli/ingress-controller/main.go

```diff
@@ -392,14 +404,44 @@ func main() {
 				Client: kongClient,
 			},
 		}
+		var cert tls.Certificate
+		if cliConfig.AdmissionWebhookCertPath != "" && cliConfig.AdmissionWebhookCert != "" {
+			if cliConfig.AdmissionWebhookCertPath != "/admission-webhook/tls.crt" {
```

Comment:
The code is checking CertPath with "" and then "/admission-webhook/tls.crt`, which is redundant.
You can simplify this code for readability.